app/src/main/java/org/piramalswasthya/cho/ui/commons/fhir_visit_details/VisitDetailAdapter.kt

```
@@ -52,22 +49,115 @@ class VisitDetailAdapter(
+                } else {
+                    itemData.chiefComplaint = ""
+                    chiefComplaintOptionInput.apply {
+                        requestFocus()
+                        boxStrokeColor = Color.RED
+                        hintTextColor = ColorStateList.valueOf(Color.RED)
+                    }
+                }
+            }
+
+            durationInput.addTextChangedListener(object : TextWatcher {
+                override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)